src/intfuncs.c

```diff
@@ -307,10 +307,18 @@ static inline uint64_t rotl64 ( uint64_t x, int8_t r )
 //-----------------------------------------------------------------------------
 // Block read - if your platform needs to do endian-swapping or can only
 // handle aligned reads, do the conversion here
+//
+// The pointer p may not be aligned, which means that directly writing to it can 
```

Comment:
very picky, it's "reading it", not "writing to it"

Reply:
OK, fixed.